.github/workflows/build_pypi_package.yaml

```diff
@@ -22,27 +21,22 @@ jobs:
   build-package:
     name: "Build & verify package"
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        directory:
-          - "conda-store"
-          - "conda-store-server"
     defaults:
       run:
-        working-directory: ${{ matrix.directory }}
+        working-directory: conda-store-server
     steps:
       - name: "Checkout Repository 🛎"
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - run: echo "Running on ${{ matrix.directory }}"
+      - run: echo "Running on conda-store-server"
```

Comment:
This is now redundant, as it was only used to check it was actually building one of the sub-packages from the correct dir. Let's remove it.